packages/core/install-package/src/validateModuleSpecifiers.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+const MODULE_REGEX = /^((@[^/\s]+\/){0,1}([^/\s.~]+[^/\s]*)){1}(@[^/\s]+){0,1}/;
```

Comment:
You have `~`, but no matching case in the test. I have never seen a `~` in a module, is that a special character to allow for `-`?

Reply:
I'll add a test, but it's mainly to disallow Parcel from trying to install local packages

Reply:
Ow there's already a test for it

Reply:
Oh I missed that. Cool!